systems/plants/CMakeLists.txt

```diff
@@ -63,10 +63,13 @@ if (eigen3_FOUND)
   add_rbm_mex(getCMMmex)
   add_rbm_mex(findKinematicPathmex)
   add_rbm_mex(geometricJacobianmex)
+  add_rbm_mex(doKinematicsNewmex)
   add_rbm_mex(forwardKinVmex)
   add_rbm_mex(forwardKinPositionGradientmex)
   add_rbm_mex(centerOfMassmex)
   add_rbm_mex(centroidalMomentumMatrixmex)
+  add_rbm_mex(massMatrixmex)
```

Comment:
I suppose compile time is a valid argument, and I will be adding a couple more. I will also remove 5 of these after the switch.
I guess I'm actually OK with it, as long as the single entry function's only job is to switch on the function name input and call the current mex functions (which will no longer be compiled as mex functions). I'm just opposed to having a single, 2000 line file that contains the method name switch as well as the and the input and output processing and RigidBodyManipulator call.
I propose not to incorporate it into this pull request though.